src/java.base/share/classes/sun/net/ftp/impl/FtpClient.java

```diff
@@ -551,6 +547,7 @@ private void issueCommandCheck(String cmd) throws sun.net.ftp.FtpProtocolExcepti
      * @return the connected <code>Socket</code>
      * @throws IOException if the connection was unsuccessful.
      */
+    @SuppressWarnings("removal")
```

Comment:
Could the scope of the annotation be further reduced by applying it to the two places where `doPrivileged` is called in this method?

Reply:
I'll probably need to assign the doPriv result on L631 to a tmp variable and then assign it to `s`. Are you OK with this ugliness?